### DISCUSSION TITLE
[@types/bootstrap-fileinput] missing browseOnZoneClick

### DISCUSSION THREAD

**Discussion by khoait** [DefinitelyTyped/DefinitelyTyped#58925] [Issues with a @types package] [answered]
Hi, I'm using bootstrap-fileinput npm package v5.2.7 in my typescript project. @types/bootstrap-fileinput is missing browseOnZoneClick option.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
fixing here: #58931 58931

**Reply by khoait** [reply]
thanks for super quick fix! 👍
